src/runtime-rs/crates/shim-ctl/README.md

```diff
@@ -0,0 +1,50 @@
+### Purpose
+shim-ctl is a binary to exercise the shim proper without containerd
+dependencies.
```

Comment:
Can we merge it into [kata-ctl](https://github.com/kata-containers/kata-containers/blob/d3ee8d9f1bcafe85991e98ea75bb9a941e8d97ef/src/tools/kata-ctl/README.md) so that we have a unified command line tool for the rust shim?

Reply:
Not required for this PR (as it has already been reviewed several rounds and is in good shape as is). But I'd like to get folks on the same page if we will merge shim-ctl into kata-ctl. /cc @jodh-intel

Reply:
I'm perfectly fine to do that.  I'd wait though to first see how this pans out. I'm still unsure if the approach taken in shim-ctl is even workable. Getting rid of the dependency on containerd is great and has enabled me to work on this so far but it also means that shim-ctl has to do some of the stuff normally done by containerd. I've no idea so far how workable this is - in the worst case it might mean duplicating of lot of containerd functionality in shim-ctl which might not be worth the trouble in the end.

So my suggestion would be to wait a bit until the approach is proven, then merge it into kata-ctl.

Reply:
Sounds good to me. Thanks @pmores !